sdk/batch/azure-compute-batch/src/main/java/com/azure/compute/batch/models/GetBatchNodeRemoteLoginSettings.java

```diff
@@ -0,0 +1,10 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+package com.azure.compute.batch.models;
+
+/**
+ * Optional parameters for getting the settings required for remote login to a Batch Compute Node.
+ */
+public class GetBatchNodeRemoteLoginSettings extends BatchBaseOptions {
```

Comment:
Shouldn't end with `Options`? If `GetBatchNodeRemoteLoginSettingsOptions` is too unwieldy maybe just `GetBatchNodeRemoteLoginOptions`?

Reply:
I guess I slightly prefer the former even though it's pretty odd-sounding... it is consistent at least.